sources/background_scripts/notifications.js

```diff
@@ -0,0 +1,25 @@
+NewTabTweaker = {};
+NewTabTweaker.Notifications = {
+	notification_ids: [],
+	notify: function(notification_id, message){
```

Comment:
Add a short documentation comment for this function, i.e. what is does, the fact it doesn't display two messages with the same identifier at the same time, and that messages are removed after a delay.

`notification_id` can be shortened to `id`. It is clear from the function and module it is attached to it relates to notifications.

Move brace to a new line as per existing code style.